commit d9a2cf229712b2b2970ce496663521cf9f5f7873
Author: Neil Horman <nhorman@tuxdriver.com>
Date:   Mon Jan 27 07:08:32 2014 -0500

    Set default deepestcache to 2
    
    Been meaning to do this for a while.  By default having a deepestcache value of
    ULONG_MAX causes irqbalance to always find the deepest cache level in a systems
    which causes it on some systems to think that all cpus share a cache, and as
    such, that no balancing is needed.  Rectify this such that the default deepest
    cache level is defaulted to 2
    
    Signed-off-by: Neil Horman <nhorman@tuxdriver.com>

diff --git a/irqbalance.1 b/irqbalance.1
index 2610cfa..f5c4567 100644
--- a/irqbalance.1
+++ b/irqbalance.1
@@ -80,12 +80,15 @@ example to ban irqs 43 and 44 from balancing, use the following command line:
 .TP
 .B --deepestcache=<integer>
 This allows a user to specify the cache level at which irqbalance partitions
-cache domains. Without specified, irqbalance searches the available deepest
-cache. This can affect how irqbalance builds up the CPU tree. For example, on
-a system where all the CPU cores being within the same L3 cache domain, one
-can let irqbalance build up the CPU tree on L2 cache by using the following
-command line: 
+cache domains.  Specifying a deeper cache may allow a greater degree of
+flexibility for irqbalance to assign irq affinity to achieve greater performance
+increases, but setting a cache depth too large on some systems (specifically
+where all cpus on a system share the deepest cache lavel), will cause irqbalance
+to see balancing as unnesessecary. 
 .B irqbalance --deepestcache=2
+.P
+The default value for deepestcache is 2.
+
 
 .TP
 .B -l, --policyscript=<script>
diff --git a/irqbalance.c b/irqbalance.c
index 2b79eac..2532242 100644
--- a/irqbalance.c
+++ b/irqbalance.c
@@ -50,7 +50,7 @@ int need_rescan;
 unsigned int log_mask = TO_ALL;
 enum hp_e hint_policy = HINT_POLICY_IGNORE;
 unsigned long power_thresh = ULONG_MAX;
-unsigned long deepest_cache = ULONG_MAX;
+unsigned long deepest_cache = 2;
 unsigned long long cycle_count = 0;
 char *pidfile = NULL;
 char *banscript = NULL;
